src/app/loadout/loadout-ui/Subclass.tsx

```diff
@@ -0,0 +1,106 @@
+import { D2ManifestDefinitions } from 'app/destiny2/d2-definitions';
+import { t } from 'app/i18next-t';
+import ConnectedInventoryItem from 'app/inventory/ConnectedInventoryItem';
+import { PluggableInventoryItemDefinition } from 'app/inventory/item-types';
+import ItemPopupTrigger from 'app/inventory/ItemPopupTrigger';
+import { isPluggableItem } from 'app/inventory/store/sockets';
+import { DimLoadoutItem } from 'app/loadout-drawer/loadout-types';
+import { AppIcon, powerActionIcon } from 'app/shell/icons';
+import { getSocketsByIndexes } from 'app/utils/socket-utils';
+import { SocketCategoryHashes } from 'data/d2/generated-enums';
+import React, { useMemo } from 'react';
+import { createGetModRenderKey } from '../mod-utils';
+import PlugDef from './PlugDef';
+import styles from './Subclass.m.scss';
+
+/** The subclass section used in the loadouts page and drawer */
+export default function Subclass({
```

Comment:
I would like a better name for this but had nothing really at the time. Best I could think of was `SubclassSection` but its not really descriptive.

Reply:
I'm generally down with longer names like `LoadoutSubclassSection`